Listenbrainz/model/Track.swift

```diff
@@ -13,3 +13,126 @@ struct Track: Codable {
     let name: String
     let duration: Int
 }
+
+// This file was generated from JSON Schema using quicktype, do not modify it directly.
+// To parse the JSON, add this file to your project and do:
+//
+//   let album = try? JSONDecoder().decode(Album.self, from: jsonData)
+//   let track = try? JSONDecoder().decode(Track.self, from: jsonData)
+
+//import Foundation
```

Comment:
Why did you comment out this file?

Reply:
I was using it for matching the response, removed it.